<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QMainWindow" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralDoctorMainwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>751</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="SearchPatientTab">
     <attribute name="title">
      <string>SearchPatient</string>
     </attribute>
     <widget class="QTextBrowser" name="TitleText">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI';&quot;&gt;Hello, doctor, please input patient's information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="LastNameLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="FirstNameLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="SSNLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SSN Last 4 #</string>
      </property>
     </widget>
     <widget class="QLabel" name="DOBLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LastNameLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="FirstNameLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="SSNLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="DOBLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTableView" name="PatientQueryTable">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>281</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="searchResultLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AddDoctorTab">
     <attribute name="title">
      <string>AddPatient</string>
     </attribute>
     <widget class="QLabel" name="HospitalID">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>HospitalID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Language">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
     <widget class="QLabel" name="Organization">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Organization</string>
      </property>
     </widget>
     <widget class="QLabel" name="ProviderID">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ProviderID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Race">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
     <widget class="QLabel" name="Religious_Affirm">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Religious Affiliation</string>
      </property>
     </widget>
     <widget class="QLabel" name="Roles">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Roles</string>
      </property>
     </widget>
     <widget class="QLabel" name="Speciality">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Speciality</string>
      </property>
     </widget>
     <widget class="QLabel" name="SubSpeciality">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SubSpeciality</string>
      </property>
     </widget>
     <widget class="QLabel" name="Type">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserID">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UserID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Family_Name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Family Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="Given_Name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Given Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hospitalID">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="language">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="organization">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="race">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="providerid">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="religiousaffirm">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="roles">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="speciality">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="subspeciality">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Type_edit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="userid">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="familyname">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="givenname">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addDoctorSaveButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="Martial_Status">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Martial Status</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="martialstatus">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="middlename_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>490</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="middlename">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Middle Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="prefixname">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prefix Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="SUFFIXNAME">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Suffix Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="HOSPITALNAME">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hospital Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="ADDRESSCOUNTRY">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address Country</string>
      </property>
     </widget>
     <widget class="QLabel" name="ADDRESSZIPCODE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address Zip Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="ADDRESS_STATE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address State</string>
      </property>
     </widget>
     <widget class="QLabel" name="ADDRESSCITY">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address City</string>
      </property>
     </widget>
     <widget class="QLabel" name="ADDRESSLINE1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address Line 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="AddressLine2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address line 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="EmailID">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>EmailID</string>
      </property>
     </widget>
     <widget class="QLabel" name="officephone">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Office Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="MOBILEPHONE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mobile Phone Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prefixname_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="suffixname">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="hospitalname">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addresscountry">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addresszipcode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_state">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addressstate">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_line1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addressline2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailid">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="office_phone">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mobilephone">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="LogOutButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDoctor">
    <property name="title">
     <string>Doctor</string>
    </property>
   </widget>
   <addaction name="menuDoctor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
